boards/x86/gpmrb/doc/index.rst

```diff
@@ -0,0 +1,133 @@
+.. _gpmrb:
+
+Gordon Peak MRB
+###############
+
+Overview
+********
+
+The Intel Gordon Peak Module Reference Board (GP MRB) is used in
+the automotive industry for the development of in-vehicle applications
+like heads-up displays and entertainment systems.
+
+.. figure:: img/gpmrb.jpg
+   :width: 500px
+   :align: center
+   :alt: Gordon Peak MRB
+
+   Gordon Peak MRB
+
+Hardware
+********
+
+.. include:: ../../../../soc/x86/apollo_lake/doc/supported_features.rst
+
+Building and Running Zephyr
+***************************
+
+Use the following procedure to boot a Zephyr application on the Gordon Peak
+MRB.
+
+Build Zephyr Application
+========================
+
+Build a Zephyr application; for instance, to build the ``hello_world``
+application for the GP MRB:
+
+   .. zephyr-app-commands::
+      :zephyr-app: samples/hello_world
+      :tool: all
+      :board: gpmrb
+      :goals: build
+
+.. note::
+
+   This will result in an output file named :file:`zephyr.elf`, which
+   is the application built as a standard ELF binary. There will also
+   be a file :file:`zephyr.strip` which is the same binary with the
+   debugging information removed. Because this binary is being burned
+   into a limited firmware flash area, we will use the smaller,
+   stripped version.
```

Comment:
Maybe this could be turned into part of the main text instead of being a note. Think some people were trying to get rid of some `note`s earlier.

Reply:
Done.